content/en/docs/resources/guides/extending-jx/_index.md

```diff
@@ -2,5 +2,5 @@
 title: "Extending Jenkins X"
 linkTitle: "Extending Jenkins X"
 description: Learn how to extend Jenkins X using the Apps Framework
-weight: 5
+weight: 12
 ---
```

Comment:
@MarckK shouldn't we add an alias pointing back to its old location to avoid 404 responses for old links that may be out there and a corresponding reduction in Google's rank?!

```
aliases:
  - /docs/guides/extending-jx/
```

Reply:
OK, have added an additional reference to each of the moved pages. 

These aliases are beginning to accumulate! :)